lib/dot/definitions.def

```diff
@@ -3,7 +3,7 @@
     var $lvl = it.level;
     var $dataLvl = it.dataLevel;
     var $schema = it.schema[$keyword];
-    var $schemaPath = it.schemaPath + '.' + $keyword;
```

Comment:
it should be: `it.schemaPath + it.util.getProperty($keyword);`

This way it would still have direct property access when possible (which is in most cases and for all standard keywords)